uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/manage-leave-requests.js

```diff
@@ -50,14 +50,20 @@ define([
         pending_requests: false,
         contact_id: null,
         selectedPeriod: null,
-        selectedAbsenceTypes: null
+        selectedAbsenceTypes: null,
+        assignedTo: 'all'
       }
     };
     vm.filteredUsers = [];
+    vm.filtersByAssignee = [
+      { type: 'all', label: 'All' },
+      { type: 'unassigned', label: 'Unassigned' },
+      { type: 'me', label: 'Assigned To Me' }
+    ];
     vm.isFilterExpanded = false;
+    vm.isAdmin = false; // this property is updated on controller initialization
```

Comment:
I am not sure about that as if you look at https://compucorp.atlassian.net/wiki/display/PCHR/Angular+1+Style+Guide+discussion there is a rule about the var types and the alphabetical order. However, I am not sure I see a rule for multiline assignments.